main.rb

```diff
@@ -0,0 +1,168 @@
+# Imports
+require './book'
+require './student'
+require './teacher'
+require './rental'
+
+# Welcome note
+puts "---------------------------------\n
+ Welcome to OOP School Library App!\n
+ ------------------------------"
+
+# State Helpers
+@book_arr = []
+@person_arr = []
+
+# Method to add new book record
+def add_book(book)
+  @book_arr.push(book)
+  @book_arr
+end
+
+# Method to add new book record
+def add_person(person)
+  @person_arr.push(person)
+  @person_arr
+end
+
+# Method to create new student's record
+def create_student
+  print 'Age: '
+  age = gets.chomp
+  print 'Name: '
+  name = gets.chomp
+  print 'Has parent permission? [Y/N]: '
+  perm_input = gets.chomp
+  case perm_input.upcase
+  when 'Y'
+    permission = true
+  when 'N'
+    permission = false
+  else
+    puts 'Invalid entry!'
+    init
+  end
+  student = Student.new(age, name)
+  student.parent_permission = permission
+  student.type = 'Student'
+  add_person(student)
+  success('Person')
+end
```

Comment:
- [ ] I have an error while creating the person, please fix it.

![image](https://user-images.githubusercontent.com/55635977/158031001-9c9b67c5-f739-4d84-b4af-b8ab57f32937.png)